src/MercadoPago/Resources/Payment/Refunds.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace MercadoPago\Resources\Payment;
+
+/** Refunds class. */
+class Refunds
```

Comment:
um nome mais adequado não seria "Refund"? porque é um refund só
```suggestion
class Refund
```

Reply:
De acordo com a Referência de API, Payment não tem esse campo Refund(s), então retirei do projeto. Vale a pena conferir na versão final do SDK.